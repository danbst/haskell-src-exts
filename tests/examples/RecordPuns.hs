module RecordPuns where

cotile = tile {Kind.ospeedup}
