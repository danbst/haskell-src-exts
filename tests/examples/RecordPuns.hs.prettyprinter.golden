ParseFailed (SrcLoc {srcFilename = "tests/examples/RecordPuns.hs", srcLine = 3, srcColumn = 29}) "NamedFieldPuns is not enabled"
